jerry-port/default/default-date.c

```diff
@@ -101,17 +100,20 @@ double jerry_port_get_current_time (void)
 #ifdef __GNUC__
   struct timeval tv;
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
   if (gettimeofday (&tv, NULL) == 0)
+#pragma GCC diagnostic pop
   {
     return ((double) tv.tv_sec) * 1000.0 + ((double) tv.tv_usec) / 1000.0;
   }
 #endif /* __GNUC__ */
 
-#ifdef _WINDOWS
+#ifdef _WIN32
   time_t ltime;
   time (&ltime);
   return ltime * 1000;
-#endif /* _WINDOWS */
+#endif /* _WIN32 */
```

Comment:
if the `#ifdef _WIN32` check came first, followed by an `#elif defined(__GNUC__)`, would that not make the pragmas unnecessary? (I.e., even if there was a GCC on Windows -- MinGW --, the Windows-specific code would be taken.)

Reply:
Is this comment resolved? I see `#endif /* __GNUC__ */` now instead of `#elif`

Reply:
This is still the original code, not updated according to review.

Reply:
The idea is good. But I found a better solution, we can use time() on all platform, we don't need platform specific implementation at all here.

Reply:
time is seconds， not ms